src/components/commons/PageHeading.tsx

```diff
@@ -1,13 +1,35 @@
-import { Heading } from '@chakra-ui/react'
+import { Flex, Heading, useMediaQuery, VStack } from '@chakra-ui/react'
+import { ReactElement } from 'react'
 
 type Props = {
   title: string
+  children?: ReactElement
 }
 
-export const PageHeading = ({ title }: Props) => {
-  return (
-    <Heading textAlign="center" my={3}>
-      {title}
-    </Heading>
-  )
+export const PageHeading = ({ title, children }: Props) => {
+  const [largeScreen] = useMediaQuery('(min-width: 41em)', { fallback: true })
+  if (!children) {
+    return (
+      <Heading textAlign="center" my={3}>
+        {title}
+      </Heading>
+    )
+  }
+  if (largeScreen) {
+    return (
+      <Flex my={3} justify="center" alignItems="center">
+        <span style={{ visibility: 'hidden', marginRight: 'auto' }}>{children}</span>
+
+        <Heading textAlign="center">{title}</Heading>
+        <span style={{ marginLeft: 'auto' }}>{children}</span>
```

Comment:
Are you sure about duplicating children?

Reply:
It was the easiest solution for keeping the title in the middle with a button to the right